src/DataCollectors/Microsoft.TestPlatform.Extensions.EventLogCollector/Properties/AssemblyInfo.cs

```diff
@@ -22,3 +23,5 @@
 
 // The following GUID is for the ID of the typelib if this project is exposed to COM
 [assembly: Guid("4c1f0d81-67a9-4bf3-a006-615ab4a7fcd6")]
+
+[assembly: TestExtensionTypes(typeof(EventLogDataCollector))]
```

Comment:
It is better to have this custom assembly before `Guid` for clear understanding.